### DISCUSSION TITLE
Looking for tasmota friendly board with ethernet and external antenna

### DISCUSSION THREAD

**Discussion by DarthWeber** [arendst/Tasmota#22014] [General]
Hi, i am looking for a tasmota friendly board with ethernet (LAN) and Bluetooth (external antenna). I don't need POE and would prefer to work with external 5V or 12V

Anny suggestions would be appreciated.
Thanks,
Tom

**Comment by s-hadinger** [comment]
Olimex provides nice Ethernet ESP32 boards. 

https://www.olimex.com/Products/IoT/ESP32/ESP32-GATEWAY/open-source-hardware

The "EA" variant has an external antenna. The board looks quite big. 

I prefer the smaller POE version 

https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware

which exists also in "EA" version for roughly the same price

**Comment by DarthWeber** [comment]
Thanks. But can i use the Poe Version also without Poe? I prefer to avoid additional injektor and splitter

25.08.2024 09:15:01 s-hadinger ***@***.***>:

**Reply by sfromis** [reply]
My general expectation would be that PoE-capable devices would still work without being powered this way, as long as they have a power source. While I can't vouch for specific devices, I saw no trouble with a different PoE-capable ESP32 board and USB power input.

**Reply by s-hadinger** [reply]
Yes. I have the Olimex device and it works without POE. In such case it takes power from the USB port

**Comment by Jason2866** [comment]
All we know is listed here https://templates.blakadder.com/

**Reply by sfromis** [reply]
If you're looking for a "finished" device with Ethernet and external antenna, this one is basically built for your use case. No personal experience with it.
https://templates.blakadder.com/GL-S10.html

When it comes to the warning about IP101GRI, Tasmota Ethernet support was expanded some time ago to include type 1 for TLK110/IP101. That looks "pretty similar", but I can't say anything for sure.
